.github/workflows/linux-raspbian.yml

```diff
@@ -0,0 +1,140 @@
+#         - dockerimg: ubuntu:devel
+#           compiler: gcc
+#           upload: 0
+#         - dockerimg: ubuntu:devel
+#           compiler: clang
+#           upload: 0
+    steps:
+    - uses: actions/checkout@v2
+    - uses: pguyot/arm-runner-action@v2
+      id: build_image
+      with:
+        base_image: raspios_lite:2022-04-04
+    - name: Release build
+      uses: addnab/docker-run-action@v3
+      if: ${{ startsWith(github.ref, 'refs/tags/') }}
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+        registry: gcr.io
+        image: ${{ matrix.dockerimg }}
+        options: >-
+          -v ${{ github.workspace }}:/sources
+          -e GITHUB_REF=${{ env.GITHUB_REF }}
+          -e GITHUB_SERVER_URL=${{ env.GITHUB_SERVER_URL }}
+          -e GITHUB_REPOSITORY=${{ env.GITHUB_REPOSITORY }}
+          -e GITHUB_RUN_ID=${{ env.GITHUB_RUN_ID }}
+          -e QTVER=${{ env.QTVER }}
+          -e PPAQTVER=${{ env.PPAQTVER }}
+          -e APPIMAGEVER=${{ env.APPIMAGEVER }}
+          -e COMPILER=${{ matrix.compiler }}
+          -e DOCKERIMG=${{ matrix.dockerimg }}
+          -e NJOBS=${{ env.NJOBS }}
+          -e QTIFWVER=${{ env.QTIFWVER }}
+          -e UPLOAD=${{ matrix.upload }}
+        run: |
+          cd /sources
+          echo
+          echo Install dependencies
+          echo
+          chmod +x ports/ci/linux-ubuntu/install_deps.sh
+          ./ports/ci/linux-ubuntu/install_deps.sh
+          echo
+          echo Release Build
+          echo
+          chmod +x ports/ci/linux-ubuntu/build.sh
+          ./ports/ci/linux-ubuntu/build.sh
+          echo
+          echo Release Deploy
+          echo
+          chmod +x ports/ci/linux-ubuntu/deploy.sh
+          ./ports/ci/linux-ubuntu/deploy.sh
+    - name: Daily build
+      uses: addnab/docker-run-action@v3
+      if: ${{ !startsWith(github.ref, 'refs/tags/') }}
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+        registry: gcr.io
+        image: ${{ matrix.dockerimg }}
+        options: >-
+          -v ${{ github.workspace }}:/sources
+          -e GITHUB_REF=${{ env.GITHUB_REF }}
+          -e GITHUB_SERVER_URL=${{ env.GITHUB_SERVER_URL }}
+          -e GITHUB_REPOSITORY=${{ env.GITHUB_REPOSITORY }}
+          -e GITHUB_RUN_ID=${{ env.GITHUB_RUN_ID }}
+          -e QTVER=${{ env.QTVER }}
+          -e PPAQTVER=${{ env.PPAQTVER }}
+          -e APPIMAGEVER=${{ env.APPIMAGEVER }}
+          -e COMPILER=${{ matrix.compiler }}
+          -e DOCKERIMG=${{ matrix.dockerimg }}
+          -e NJOBS=${{ env.NJOBS }}
+          -e QTIFWVER=${{ env.QTIFWVER }}
+          -e UPLOAD=${{ matrix.upload }}
+          -e DAILY_BUILD=1
+        run: |
+          cd /sources
+          echo
+          echo Install dependencies
+          echo
+          chmod +x ports/ci/linux-ubuntu/install_deps.sh
+          ./ports/ci/linux-ubuntu/install_deps.sh
```

Comment:
You must create a new ```./ports/ci/linux-raspbian``` directory with the scripts there.

Reply:
Done